app/Repositories/Eloquent/BaseRepository.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Repositories\Eloquent;
+
+use App\Repositories\EloquentRespositoryInterface;
+use Illuminate\Database\Eloquent\Model;
+
+class BaseRepository
+{
+    protected Model $model;
+
+    public function __construct(Model $model)
+    {
+        $this->model = $model;
+    }
+
```

Comment:
getAll($with = [])
{
    return $this->model->newModelQuery()->with($with)->get();
}

$this->customerRepository->getAll ( ['orders', 'country' ] );